server.js

```diff
@@ -289,6 +289,15 @@ function cache(f) {
   };
 }
 
+camp.notfound(/\.(svg|png|gif|jpg|json)/, 
+  cache(function(data, match, sendBadge, request) {
```

Comment:
Filling up the cache with data which is trivially recomputed doesn't make sense.

Is there an advantage to caching which I'm missing?